Dockerfile

```diff
@@ -34,7 +34,7 @@ RUN /usr/local/bin/docker-entrypoint.sh generate \
     -c config/autogen-server.json
 
 # Base image
-FROM artifactory.algol60.net/docker.io/alpine:3.13 as base
+FROM artifactory.algol60.net/docker.io/alpine:3.15 as base
```

Comment:
Don't forget to update the copyright date...

Reply:
fixed